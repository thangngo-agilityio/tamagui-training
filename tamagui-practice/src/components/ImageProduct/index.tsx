import { Flex } from '@chakra-ui/react';
import Image from 'next/image';

type TImageProduct = {
  image: string[];
  alt: string;
};

const ImageProduct = ({ image, alt }: TImageProduct) => (
  <Flex as="div" borderRadius="sm" flex={1} flexDir="column" gap="10px">
    {image.map((link, index) => (
      <Image
        key={index}
        width={647}
        height={549}
        src={link}
        alt={alt}
        style={{ backgroundSize: '100%', borderRadius: '8px', width: '100%' }}
      />
    ))}
  </Flex>
);

export default ImageProduct;
