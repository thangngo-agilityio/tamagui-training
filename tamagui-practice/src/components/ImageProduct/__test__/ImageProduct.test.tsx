import { render } from '@testing-library/react';
import ImageProduct from '..';

describe('ImageProduct component', () => {
  it('should match snapshot for ImageProduct', () => {
    const { container } = render(<ImageProduct image={[]} alt="test" />);

    expect(container).toMatchSnapshot();
  });
});
