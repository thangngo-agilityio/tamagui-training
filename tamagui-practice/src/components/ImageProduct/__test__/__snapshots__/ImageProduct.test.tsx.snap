// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageProduct component should match snapshot for ImageProduct 1`] = `
<div>
  <div
    class="css-1sawh8z"
  />
</div>
`;
